### ISSUE TITLE
switch qwen2.5vl

### ISSUE THREAD

**Issue by gaoyixuan1025** [zhiyuanyou/DeQA-Score#26]
Thank you for your response. After verification, we found that len(level_id) == 1, level_id[0] == true_id. 
Similarly,
in softkl_loss, Qwen's tokenizer uses 6 IDs for self.config.level_prefix, indicating no BOS token. Therefore, should we remove idx_level_logit = idx_level_label - 1 and directly use idx_level_label to extract logits for predicting the level_token ID?
